tests/kuttl/common/assert_sample_deployment.yaml

```diff
@@ -0,0 +1,202 @@
+apiVersion: cinder.openstack.org/v1beta1
+kind: Cinder
+metadata:
+  name: cinder
+  namespace: openstack
+spec:
+  serviceUser: cinder
+  customServiceConfig: |
+    [DEFAULT]
+    debug = true
+  databaseInstance: openstack
+  databaseUser: cinder
+  rabbitMqClusterName: rabbitmq
+  cinderAPI:
+    replicas: 1
+    containerImage: quay.io/tripleozedcentos9/openstack-cinder-api:current-tripleo
+  cinderScheduler:
+    replicas: 1
+    containerImage: quay.io/tripleozedcentos9/openstack-cinder-scheduler:current-tripleo
+  cinderBackup:
+    replicas: 1
+    containerImage: quay.io/tripleozedcentos9/openstack-cinder-backup:current-tripleo
+  secret: cinder-secret
```

Comment:
In my environment this appears to be `osp-secret`, if it's something variable, it would be best to remove it

Reply:
ack, removing for now. In a separate PR I'll try to update this with something proper.